templates/default.nix

```diff
@@ -1,6 +1,6 @@
 # Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
 # SPDX-License-Identifier: Apache-2.0
-{self}: {
+{...}: {
```

Comment:
A nit-pick but can't we just remove the function from this altogether, i.e. the `{...}:` part, and then just remove the `{self}` from the flake. I generally don't like using this `...`-syntax because it hides things.

If we need to pass self later we can then add it again.